applications/schedview-snapshot/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
We generally have not been using .helmignore files in our applications.  Is there something in particular that's been choking if you try to do a `helm template` on it in your development environment?